### DISCUSSION TITLE
Weekly Feedback #1.2 - Defining Our MVP

### DISCUSSION THREAD

**Discussion by ajbott** [PullDating/Org#205] [General]
## [MVP Definitions Wiki Page](https://github.com/PullDating/Org/wiki/MVP-Definition)

We're on the lookout for insights and ideas to shape the core of our Minimum Viable Product (MVP). Your contributions are crucial in steering the direction of our development. Dive into the [Wiki page](https://github.com/PullDating/Org/wiki/MVP-Definition) to catch up on our current progress. If something sparks an idea or if you have features in mind that you believe should be part of our MVP, we'd love to hear from you! Understanding what's essential from the get-go enables us to lay a solid foundation and map out a clear path to achieving our vision. Let's collaborate to make this app not just a product, but a journey crafted by and for our community.

### User Needs and Expectations:
- What are the most essential features you look for in a dating app?
- Can you describe a feature you've encountered in other dating apps that you felt was unnecessary or complicated the experience?

### Safety and Security:
- What safety features are most important to you in a dating app?
- Have you ever had a safety concern while using a dating app, and how do you wish it was addressed?

### User Interface and Experience:
- What has been your experience with setting up a profile on dating apps? What did you like or dislike about the process?
- How do you prefer to interact with potential matches? (e.g., swiping, liking, messaging)

### Matching and Filtering:
- How important are filtering options (like age, distance, interests) to you in finding matches?
- What do you think about the idea of seeing only users who meet your preferences and whose preferences you also meet?

### Community Engagement:
- Would you value features that allow for community feedback or participation in the app's development? How might you want to contribute?
- What kind of community-driven features would make a dating app more appealing to you?

### Privacy and Data Use:
- How do you feel about the way dating apps use your personal data? Are there specific practices you appreciate or find concerning?
- What level of control do you expect to have over your personal information on a dating app?

### Unique Value Proposition:
- What is missing from current dating apps that you wish existed?
- If there was one thing that could make a dating app stand out for you, what would it be?

### Feedback and Improvement:
- How likely are you to use a feature that lets you give direct feedback to the app developers?
- In what ways would you like to see your feedback impact the development of the app?

### General Questions:
- Describe a feature you loved in another app and why it was important to you.
- What frustrations have you encountered with profile setups?
- What feature(s) of a dating app do you think are the most important?
- How do you like to showcase your personality and interests in your dating profile?
- What features in a dating app make you feel safe and why?

**Comment by Oshyan** [comment]
I am interested in talking about almost all of these things, but I also don't think we should treat this as an entirely open, unclear problem. I note that the MVP definitions page that is referenced makes the same point, but it feels like this page then does start to open things up to the possibility of reinvention or design by committee. IMO don't need 1000 opinions on why dating apps suck or how to build the best one. There will be lots of individual variation, and the impact and importance of most identified problems and features will pale in comparison to what in my view is **the** problem with essentially every dating app: the profit motive and the need to pay for features. Critically the profit motive doesn't only drive the actual paywalled features, it also deeply influences app design! Tinder's swipe model, much like TikTok's, is designed for maximum engagement, not for the actual match finding needs and behavior of real, emotion-having humans.

So the first step to figuring out what an MVP might look at to me is to try to imagine existing, otherwise good apps with all of their paid features being free. Would that meet most user's needs well? If not, why not? No need to fully reinvent the wheel here. I also think it would probably be ideal to get one or more related-focus psychologists and/or therapists (or relationship coaches, etc.) involved, if possible. People with actual psychological and emotional expertise who can speak to what kinds of incentives and experiences different sets of features, match browsing models, etc. might create and the outcomes that might contribute to.

That said, imagining apps without a profit motive can be instructive, but it doesn't actually solve the very real-world problem of revenue! So I think that is a key component of any "MVP" consideration, even though it's not a technical one. What is "minimum **viable**" (emphasis on the "viable") is what is long-term sustainable, i.e. we must solve the revenue problem!

As far as examples of specific apps that might be inspiring, for my part - even though I'm not particularly kink-oriented - I actually think Feeld is an excellent reference point. Its "swipe" is really just linear profile browsing. You can freely browse back and forth and linger or go back to anyone you want. You can filter by what users of the app theoretically care most about, what kinds of relationships they want, and kink interests. I think a more broad appeal app would also let you filter by "interests" or other lifestyle stuff like vegetarian vs. meat eater, drinking vs. non, travel-oriented vs. homebody, whatever. This is to some degree where "quizzes" *can* come into play, but they are not necessarily the most effective solution. On that note Firefly is another reference point to consider, and it's currently entirely free, but it's a bit clunky and really just trying to be like OKC of old (which is not a bad idea in many respects, because OKC used to be pretty good). 

Lastly I would mention that the _culture_ of an app is actually a pretty big deal, and again heavily influences how it feels to use it, and what kinds of interactions are possible and even likely/encouraged on it. Feeld is again a great example. To me it feels _notably_ different reading people's profiles there vs. most other places, with the possible exception of OKC, although there I would say there is not to much openness per se. And that, I think, is one of the things that's going on: because Feeld is a kink-oriented app, it inherently involves or at least invites a lot more openness and candor, which is remarkably refreshing. Clear communication and consent is also somewhat built-in to the kink, poly, "open" community in general. And to be clear this is something that I, as a not-actually-kinky person am really enjoying and finding refreshing, but not everyone would. 😄 I bring it up more just to illustrate how important the "culture" of an app can be. OKC of old vs. Tinder, Hinge, and others with short profile prompts and character limits also seem, in my experience, to have a notable impact not just on what people write about themselves, but also on their subsequent communications if you match, whether that's due to self-selection (i.e. people who tend to be less loquacious are more comfortable on dating apps with character limits in profiles), or because the actual context and feel of the app essentially communicates to people that short and "quippy" is the way to go.
